cockatrice/src/dlg_update.cpp

```diff
@@ -142,6 +147,12 @@ void DlgUpdate::enableUpdateButton(bool enable) {
     manualDownload->setEnabled(enable);
 }
 
+void DlgUpdate::addStopDownloadAndRemoveOthers(bool enable) {
+    enable ? buttonBox->addButton(stopDownload, QDialogButtonBox::ActionRole) : buttonBox->removeButton(stopDownload);
```

Comment:
Don't use ternary expressions as statements; only use them when you're using the value that the ternary returns. Use an explicit if/else here